libraries/gpu-gl-common/src/gpu/gl/GLBackend.cpp

```diff
@@ -153,7 +153,7 @@ void GLBackend::init() {
 
         if (vendor.contains("NVIDIA") ) {
             qCDebug(gpugllogging) << "NVIDIA card detected";
-#if !defined(Q_OS_ANDROID)
+#if !defined(Q_OS_ANDROID) == !defined(USE_GLES)
```

Comment:
should these all be &&?

Reply:
I guess so. Doesn't seem like they need to be, but I guess it is good practice.